errors.toml

```diff
@@ -2231,6 +2231,11 @@ error = '''
 User %s already exists.
 '''
 
+["schema:3750"]
+error = '''
+Unable to create or change a table without a primary key, when the system variable 'sql_require_primary_key' is set. Add a primary key to the table or unset this variable to avoid this message. Note that tables without a primary key can cause performance problems in row-based replication, so please consult your DBA before changing this setting.
```

Comment:
This is the same error mysql returns, copied from [here](https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html#:~:text=added%20in%208.0.13.-,Error%20number%3A%203750,-%3B%20Symbol%3A%20ER_TABLE_WITHOUT_PK)